app/views/content/train-to-be-a-teacher/initial-teacher-training.md

```diff
@@ -33,7 +29,7 @@ You'll spend most of your time in school placements with some time dedicated to
 * tasks and assignments
 * engagement with academic or professional research
 
-The theoretical part of your training might be delivered by a university or in a school. Some training courses will be all school-based, but this does not mean that you’ll spend all your time teaching.
+The theoretical part of your training might be delivered by a university or in a school. Some training courses will be all school-based, but this does not mean that you’ll spend all your time teaching. Find out [how to choose your teacher training course](/train-to-be-a-teacher/how-to-choose-your-teacher-training-course).
```

Comment:
`Find out` sounds slightly odd - we are not telling people the answer, just giving them some things to consider? But I don't feel strongly about this so ignore as needed!